common/snapshots.py

```diff
@@ -1241,11 +1242,27 @@ def takeSnapshot(self, sid, now, include_folders):
         has_errors = False  # TODO Fix inconsistent usage: Collects return value,
                             #      but errors are also checked via params[0]
 
-        if rsync_exit_code != 0:  # indicates an error
+        # dict of exit codes (as keys) that are treated as INFO only by BiT
+        # (not as ERROR). The values are message strings for the snapshot log.
+        rsync_non_error_exit_codes = {
+            0: _("Success"),
+            24: _("Partial transfer due to vanished source files (see 'man rsync')")
+        }
+
+        rsync_exit_code_msg = _("'rsync' ended with exit code {exit_code}".format(exit_code=rsync_exit_code))
+
+        if rsync_exit_code in rsync_non_error_exit_codes:
+            self.setTakeSnapshotMessage(0,
```

Comment:
```
self.setTakeSnapshotMessage(0,
                            rsync_exit_code_msg + _(": {msg}".format(msg=rsync_non_error_exit_codes[rsync_exit_code])))
```

I'm not sure what your goal is here. The `format()` is executed **before** the translation.
The translator on Weblate won't get this string

    ": {msg}"

but something like this if me as a maintainer with German local on his developer machine did managed the translations.

    ": Teilweise Übertragung aufgrund von verschwundenen Quelldateien (siehe 'man rsync')"

Because of line 1248 and 1249 the message is still translated and then injected into `": {msg}".

To my understanding there is no need for translation in this line. Make it like this:

```
self.setTakeSnapshotMessage(
    0,
    rsync_exit_code_msg + ": {msg}".format(msg=rsync_non_error_exit_codes[rsync_exit_code]))
```

Reply:
No I do see it too, will fix this :-)